client/components/token-field/index.jsx

```diff
@@ -183,6 +183,12 @@ var TokenField = React.createClass( {
 			} );
 		}
 
+		if ( autoCapitalize ) {
+			Object.assign( props, {
+				autoCapitalize: autoCapitalize
+			} );
```

Comment:
Yeah, I was wondering about that. Line 181 did it that way and so I thought I'd stay consistent. We should probably update that too.